src/w3c/informative.js

```diff
@@ -1,10 +1,13 @@
 // Module w3c/informative
 // Mark specific sections as informative, based on CSS
+import "deps/hyperhtml";
 export const name = "w3c/informative";
 
-export function run(conf, doc, cb) {
-  $("section.informative")
-    .find("> h2:first, > h3:first, > h4:first, > h5:first, > h6:first")
-    .after("<p><em>This section is non-normative.</em></p>");
-  cb();
+export function run() {
+  Array.from(document.querySelectorAll("section.informative"))
+    .map(informative => Array.from(informative.children).find(child => child instanceof HTMLHeadingElement))
```

Comment:
Nice!

Reply:
I’m now kinda unsure if we shouldn’t let a selector find the headings. In addition to the array creation, we are walking potentially all the children per section, which seems a little excessive. Wdyt?

Reply:
Hmm yes, that may be a problem, although I think an informative section should only contain ~10 direct descendants.

Reply:
My drive was that the original jQuery selector doesn't have a widely available standard equivalent.

```
> h2:first, > h3:first, > h4:first, > h5:first, > h6:first
```

We can just ignore `:first` part (which forces the selector to stop at the first match) but not `>`, which limits the selector for the direct descendants and has a non-widely available equivalent [`:scope`](https://developer.mozilla.org/en-US/docs/Web/CSS/:scope).

But I'm starting to think that we probably also can ignore `>` as anyway we shouldn't see an indirect child headings. Wdyt?

Reply:
All sections are supposed to have a heading (and ReSpec complains if they don't)... so, yeah, that could work. Let's try it.